### ISSUE TITLE
Setting up 3 slash commands caused cloudflare ban

### ISSUE THREAD

**Issue by FightMan01** [nextcord/nextcord#310]
### Summary

I tried to set up 3 slash commands with a bot over 11k guilds

### Reproduction Steps

Set up multiple slash comands.

### Expected Results

Slash commands will be created.

### Actual Results

I've got a global rate limit then a cloudflare ban.
I have a log and also I log all the rate limits into my console. My console was spammed with this:
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/29693718/147405231-ff9aaf7e-17a0-4c3a-a5b9-83b897a458dc.png">
I have 21k lines of this :/

### Intents

all

### System Information

- Python v3.9.2-final
- nextcord v2.0.0-alpha
    - nextcord pkg_resources: v2.0.0a4
- aiohttp v3.8.1
- system info: Linux 5.10.0-9-amd64 #1 SMP Debian 5.10.70-1 (2021-09-30)

### Checklist

- [X] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [X] I have removed my token from display, if visible.

**Comment by ZRunner** [comment]
Hey, would you share the code you used to registrer your commands please? Just to make sure you did absolutely everything right (also it may help debugging the code, who knows)

**Comment by alentoghostflame** [comment]
As part of synchronization between the bot and Discord, one web call is made per guild on startup. Across 11k guilds, this behavior is obviously not helpful. To prevent the web calls from being made, please override the `on_guild_available` event, as that's where the calls are being made.

Be warned, this will prevent the bot from syncing and deleting unknown or adding new commands across guilds. If you want to have it sync with a specific guild, use `get_guild` or something similar to get the `Guild` object, then `await guild.deploy_application_commands()` and `await guild.register_new_application_commands()`.

A toggle to make the bot not check for existing commands when no commands are added to them is coming soon.

**Comment by FightMan01** [comment]
I had a handmade slash handler when nextcord didn't have one. 
I just used the `https://discord.com/api/v9/applications/{bot_user_id_here}/commands` endpoint to register a slash command globally. This also synced new commands and the changes in the existing commands when I needed. For me needs (and I think the need for most of us) this is enough.
I don't see the point why nextcord goes through each guild and get the existing slash commands.

To @ZRunner :
```py
    @slash_command(description="A bot prefixének visszaállítása.", name="prefix")
    async def _prefix_slash(self, interaction: Interaction):
        ctx = await self.client.get_context(interaction.message)
        textcontent = await self._prefix(ctx)
        await interaction.response.send_message(embed=textcontent)
```

**Comment by alentoghostflame** [comment]
> I don't see the point why nextcord goes through each guild and get the existing slash commands.

It was so the bot could figure out if it needed to delete or register commands to a specific guild. This behavior will be changed in the next wave of slash fixes so that it won't check guilds that don't have commands trying to roll out to them, unless Client/Bot has a specific kwarg enabling the the behavior that is currently current.

**Comment by ooliver1** [comment]
@FightMan01 does this still occur after updating to latest on github

**Comment by vcokltfre** [comment]
Fixed by #320
